From d863ce14300576c53492713140eb0dfacba1d8ab Mon Sep 17 00:00:00 2001
From: Samuli Piippo <samuli.piippo@qt.io>
Date: Fri, 28 Sep 2018 13:30:56 +0300
Subject: [PATCH] Disable more warnings

Current compiler options are more strict than the old kernel is
prepared. Disable those warnings.
---
 Makefile | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/Makefile b/Makefile
index 43a6127..d3e40f3 100644
--- a/Makefile
+++ b/Makefile
@@ -627,6 +627,11 @@ ARCH_AFLAGS :=
 ARCH_CFLAGS :=
 include arch/$(SRCARCH)/Makefile
 
+KBUILD_CFLAGS	+= $(call cc-disable-warning,format-overflow,)
+KBUILD_CFLAGS	+= $(call cc-disable-warning,format-truncation,)
+KBUILD_CFLAGS	+= $(call cc-disable-warning,int-in-bool-context,)
+KBUILD_CFLAGS	+= $(call cc-disable-warning,misleading-indentation,)
+KBUILD_CFLAGS	+= $(call cc-disable-warning,tautological-compare,)
 KBUILD_CFLAGS	+= $(call cc-option,-fno-delete-null-pointer-checks,)
 KBUILD_CFLAGS	+= $(call cc-disable-warning,frame-address,)
 
